styles.css

```diff
@@ -0,0 +1,68 @@
+:root {
+  --white-color: white;
+}
+
+.body {
+  background-color: #1c1a19;
+  margin: 0;
+}
+
+.header {
+  padding: 0.5em;
+  background-color: #3c3a39;
+  position: fixed;
+  width: 99%;
+}
```

Comment:
- [ ] You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) The hamburger button is on the right, with the margins (top, bottom, left, right) defined in the design. Please try to adjust hamburger button alignment. 

The Figma design Look Likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/191768824-4f8ee8e1-354d-4457-b0ad-ffe7bdd852ca.PNG)

Your Project Look Likes this : 
![ca1](https://user-images.githubusercontent.com/93598189/191768905-fb056bf9-521e-452b-aee0-50165a6fcaeb.PNG)